tests/integration_tests/build/test_binary_size.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+"""Tests that check if the release binary sizes fall within expected size"""
+
+import os
+import pytest
+import platform
+
+import host_tools.cargo_build as host
+
+MACHINE = platform.machine()
+""" Platform definition used to select the correct size target"""
+
+FC_BINARY_SIZE_TARGET = 3137280 if MACHINE == "x86_64" else 3536544
```

Comment:
Interesting how it's significantly bigger on `aarch64`.

Reply:
Wouldn't a map per arch/features look better?

Reply:
> Interesting how it's significantly bigger on `aarch64`.

Off the top of my head, a bigger binary for aarch64 could have as its source the static linking to the libfdt library which enables the flatten device tree construction.